templates/default/activiti-app.properties.erb

```diff
@@ -1,6 +1,6 @@
 <% if @properties %>
   <% @properties.each do |property_type, value| %>
-    <% if value %>
+    <% if !value.nil? %>
```

Comment:
in this case, is better if you use !value.empty? 
That's because nil is an object in ruby, and that is different than empty , plus is more appropriate to treat strings with proper string-related methods!